Packs/EDL/Integrations/EDL/EDL.py

```diff
@@ -216,14 +217,51 @@ def ips_to_ranges(ips: Iterable, collapse_ips: str):
     Returns:
         Set. a list to Ranges or CIDRs.
     """
+    invalid_ips = []
+    valid_ips = []
 
-    if collapse_ips == COLLAPSE_TO_RANGES:
-        ips_range_groups = IPSet(ips).iter_ipranges()
-        return ip_groups_to_ranges(ips_range_groups)
+    for ip in ips:
+        if is_valid_ip(ip):
+            valid_ips.append(ip)
+        else:
+            invalid_ips.append(ip)
 
+    if collapse_ips == COLLAPSE_TO_RANGES:
+        ips_range_groups = IPSet(valid_ips).iter_ipranges()
+        collapsed_list = ip_groups_to_ranges(ips_range_groups)
     else:
-        cidrs = IPSet(ips).iter_cidrs()
-        return ip_groups_to_cidrs(cidrs)
+        cidrs = IPSet(valid_ips).iter_cidrs()
+        collapsed_list = ip_groups_to_cidrs(cidrs)
+
+    collapsed_list.update(invalid_ips)
```

Comment:
Why do we still want to add the invalid ips?

Reply:
It's a valid usecase. Could be a valid indicator that was created with a wrong type, so it should be in the EDL, but not calculated under collapsible ips.